x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/execution_python_tty_shell.json

```diff
@@ -43,5 +43,5 @@
   "timeline_title": "Comprehensive Process Timeline",
   "timestamp_override": "event.ingested",
   "type": "query",
-  "version": 7
+  "version": 9
```

Comment:
This is ok - from the manually reconciled version bumps elastic/detection-rules/pull/1950